src/components/panels/NewsPanel.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <HubPanelWrap :title="headertext">
+    <b-carousel id="infoscreen-carousel" :interval="9000" :indicators="false" :controls="true">
+      <b-carousel-slide
+        v-for="(newsitem,index) in news"
+        :key="index"
+        v-if="newsitem.translations['en']"
+        :text="title(newsitem)"
+        v-on:click.native="open(newsitem.link)"
+      >
+        <span slot="img" class="middling-helper"></span>
+        <b-img slot="img" class="d-block slide-image" fluid center :src="getimgling(newsitem.imageLink)" :alt="title(newsitem)" />
+      </b-carousel-slide>
+    </b-carousel>
+  </HubPanelWrap>
+</template>
+<style>
+.carousel-caption {
+  background-color: rgba(64, 64, 64, 0.7) !important;
+  color: white;
+  width: 100% !important;
+  left: 0 !important;
```

Comment:
This is a large number of `!important` uses that we may wish to clean up later.

Reply:
yeah, it's because of how the panel bitti used for it pulled in styles...